.buildconfig/macos-latest.yml

```diff
@@ -4,25 +4,7 @@ channels:
 dependencies:
   - cmake
   - conan>=1.45.0
-  - confuse
-  - graphlib-backport
-  - h5py
-  - hypothesis
-  - ipympl<0.9
-  - ipywidgets
-  - markupsafe>=1.1.1,<2.1.0
-  - matplotlib-base
-  - nbconvert=6.4.5
```

Comment:
We seem to have lost this pinning for `nbconvert`. Is that bad?

Reply:
Seems to work without pinning now?